src/components/PlotWrapper.tsx

```diff
@@ -48,5 +48,23 @@ export default function PlotWrapper(inputProps: PlotWrapperProps): ReactElement
     }
   }, [props.selectedTrack, props.featureName]);
 
-  return <div ref={plotDivRef} style={{ width: "600px", height: "400px" }} />;
+  // TODO: Troubleshoot using window.addEventListener for resizing, because
+  // the native responsive behavior can be a bit slow.
+  // Some layout issues occurred when using this, so it's disabled for now.
+  const updatePlotSize = (): void => {
+    if (!plotDivRef.current) {
+      return;
+    }
+    const width = plotDivRef.current.clientWidth;
+    const height = plotDivRef.current.clientHeight;
+    console.log(`Resizing (${width}, ${height})`);
+    plot?.setSize(width, height);
+  };
+  useEffect(() => {
+    updatePlotSize();
+    // window.addEventListener("resize", updatePlotSize);
+    // return () => window.removeEventListener("resize", updatePlotSize);
```

Comment:
why do you want the resize handlers in there? is there something not working that would work better with them?

Reply:
Yeah, resizing of the plot can lag behind the window dimensions changing by a few seconds. Having window resize event listeners would let it respond instantly to window size changes, but it seems to get stuck at the maximum width when they're enabled.

Reply:
I suggest making a ticket for that if you haven't already. That way it will at least be documented for later attention.

Reply:
Made the ticket: https://github.com/allen-cell-animated/nucmorph-colorizer/issues/131